Dockerfile

```diff
@@ -4,17 +4,13 @@ LABEL "maintainer"="Sviatoslav Sydorenko <wk+re-actors@sydorenko.org.ua>"
 LABEL "repository"="https://github.com/re-actors/gh-action-pypi-publish"
 LABEL "homepage"="https://github.com/re-actors/gh-action-pypi-publish"
 
-LABEL "com.github.actions.name"="pypi-publish"
```

Comment:
Not sure if I want to remove this just yet. Will this affect anyone possibly using the old Actions? Maybe it's better to remove it after Nov 13?

Reply:
As it is this breaks compatibility with any HCL stuff altogether due to the Dockerfile changes. Not sure if it is worth to try to keep compatibility given that it was and still is under a beta.

An option would be to merge to into a different branch, and update the readme to reflect the usage.

```
uses: pypi/gh-action-pypi-publish@v2
``` 

or so.
Or roll back some changes and change the usage from `with: ...` over to `env`, so that the same version is compatible with both. (But given how much of a hassle it is to test the changes currently due to long feedback loops I'd rather not do that). But if you'd prefer that maybe you'd like to take it from here?

Reply:
On the second thought, I probably don't care much. I guess the impact is minimal if any.